api/nodes/serializers.py

```diff
@@ -689,10 +689,13 @@ def create(self, validated_data):
                     'visible': contributor.visible,
                 })
                 if not contributor.user.is_registered:
-                    node.add_unregistered_contributor(
-                        fullname=contributor.user.fullname, email=contributor.user.email, auth=auth,
-                        permissions=parent.get_permissions(contributor.user), existing_user=contributor.user,
-                    )
+                    try:
+                        node.add_unregistered_contributor(
+                            fullname=contributor.user.fullname, email=contributor.user.email, auth=auth,
+                            permissions=parent.get_permissions(contributor.user), existing_user=contributor.user,
+                        )
+                    except ValidationError as e:
+                        raise InvalidModelValueError(detail=str(e))
```

Comment:
So this would be thrown if I'm creating a component and have specified to inherit the contributors from the parent.  So if a parent contributor has a blacklisted email (previous bad data, because we're making this more difficult), and I go to add them to the component, we'll throw an error.  So an unusual case. Still, mind making a test for this just to verify behavior?